.cargo/config.toml

```diff
@@ -0,0 +1,9 @@
+[alias]
+m = "make"
+mr = "m run"
+
+[net]
+git-fetch-with-cli = true
+
+[build]
+#rustflags=["-C", "link-arg=-fuse-ld=lld"]
```

Comment:
I'm probably lacking some context here, but is there any specific reason to keep the `[build]` section in `config.toml`?

Reply:
I left this because actually `lld` is not working with `trunk`. Maybe at one time will work and this flag could be uncommented. [Here](https://drone-1.prima.it/primait/localauth0/110/1/4) the drone step that fails

Reply:
Should be specific for the target.
```toml
[target.x86_64-unknown-linux-gnu]
rustflags=["-C", "link-arg=-fuse-ld=lld"]
```